### PR TITLE
Converted all theano.tensor T aliases to tt

### PR DESCRIPTION
**Pull request by fonnesbeck**
Did a global search and replace.

### CODE REVIEW

File: pymc3/distributions/continuous.py [hunk 1]
```diff
@@ -576,7 +576,7 @@ class StudentT(Continuous):
 
     Describes a normal variable whose precision is gamma distributed.
     If only nu parameter is passed, this specifies a standard (central)
-    Student's T.
+    Student's tt.
```
Comment (twiecki) [inline pymc3/distributions/continuous.py]: typo

**Comment by twiecki** [comment]
Looked through, LGTM.

**Comment by jamestwebber** [comment]
In the course of writing #1183 I noticed that there are some places where `theano.tensor` is used in its entirety, even though `import theano.tensor as tt` is also present in the file ([here is an example](https://github.com/pymc-devs/pymc3/blob/master/pymc3/model.py#L551)).

This is an even-higher level of pedantry than #1044 but it would be easy to create a PR that fixes this. Happy to do so if desired.

**Comment by twiecki** [comment]
Yes, definitely desired as I'm pedantic :)

**Pull request merged**